### DISCUSSION TITLE
Default sort when visualization is loaded

### DISCUSSION THREAD

**Discussion by sduenas** [fbaligand/kibana-enhanced-table#224] [Q&A]
Is it possible to sort a table by a computed column when the visualization is loaded?

I have a total column that is the aggregation of several ones (a total column). I would like that, when the visualization is loaded, data is sorted and shown by this column (ascending or descending order). I know that with normal columns is possible to do it but I haven't found the way to do it with a computed column.

Sorry if this was already answered but I couldn't find it in the documentation.

**Comment by fbaligand** [comment]
Hi,

Well, it is quite simple:
- You edit your "enhanced table" visualization
- You click on your total column to sort it (eventually 2 times to do reverse sorting).
- You click on "Save" button
- And you're done!

**Reply by sduenas** [reply]
Thanks, so easy!

(This is the right answer but github doesn't let me to mark it. Btw thanks for your amazing job!)

**Reply by fbaligand** [reply]
Nice to see that it answers your need!
